Makefile

```diff
@@ -16,10 +18,24 @@ cook-image:
 push-image:
 	docker push $(DOCKER_TAG)
 
-.PHONY: test
-test:
-	tox
+venv: requirements.txt
+	python ./vendor/venv-update venv= venv -ppython3 install= -r requirements.txt
```

Comment:
nit: I'd just make `vendor/venv-update` executable and split it across lines [like in ocflib](https://github.com/ocf/ocflib/blob/9028b7c728399d7f68a0e3aa179478b08e190783/Makefile#L2-L4), seems a little cleaner, but not a big deal either way.